scripts/animatediff_ui.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         interp_x=10,
         video_source=None,
         video_path='',
+        request_id = '1',
```

Comment:
put request_id at the bottom of the argument list. The default value should be ''